plugin/core/windows.py

```diff
@@ -522,12 +522,27 @@ class WindowRegistry:
     def __init__(self) -> None:
         self._enabled = False
         self._windows: dict[int, WindowManager] = {}
-        client_configs.set_listener(self._on_client_config_updated)
+        client_configs.set_listeners(self._on_client_config_updated, self._on_userprefs_updated)
```

Comment:
Now that we are passing two function references, it might make sense to create an interface with `on_client_config_updated` and `on_userprefs_updated` that `WindowRegistry` would inherit and we'd just pass `self` here.

(just a suggestion)

Reply:
https://github.com/sublimelsp/LSP/pull/2448/commits/bc61186f5f89a681e3b169d4b9d2309aa33120e6